config/settings.yml

```
@@ -29,3 +29,5 @@ plugins:
     default: ''
     type: list
     list_type: compact
+  openid_connect_claims:
+    default: ''
```

```suggestion
    default: ''
    textarea: true
```
This will make it a little easier to type long JSON strings